src/parse_instruction.py

```diff
@@ -280,6 +278,17 @@ def constant_fold(arg: Argument) -> Argument:
     return arg
 
 
+def detect_bare_reg(arg: Argument, arch: ArchAsmParsing) -> Argument:
+    """Check if `arg` is an AsmGlobalSymbol whose name matches a known Register"""
```

Comment:
the name/doc comment makes this sound like it returns a bool, and also doesn't communicate that it handles aliased registers